### DISCUSSION TITLE
how many users are foreigners? how many are chinese? why discuss in English?

### DISCUSSION THREAD

**Discussion by reallyreallycai** [element-plus/element-plus#12338] [Polls]
I think most users of element-plus are Chinese. Is that so?
Although most developers can speak English, and Github is an English website, I think Element‘s members can do some statistics and show us the result.

**Comment by iBobik** [comment]
Btw, seeing Chinese everywhere in issues is a big minus for me, because when I will need to search issues I will not be able to quickly evaluate if they are relevant for me.

This is reason why I stick with Vuetify on production projects, will have to try Elementor on some hobby project and why I declained Ant (where Chinese is used slightly more).

Nothing personal to Chinese people who helps a lot to this open source movement. Just offering point of view of potential user who is afraid of worse support when I will need it.

I see similarities to Czech projects (where I am from) which are popular here and want to be relevant also internationally, but failing to be because international audience is afraid to use something which seems like a local project where only part of the community is comfortable with English.
